### PR TITLE
Heal Travis CI.

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,6 +31,6 @@ before_script:
 
 script:
   - yarn run lint
-  - yarn test
+#  - yarn test
```
Comment (drets) [inline .travis.yml:34]: Commented intentionally since we don't have tests for app (yet).

**Comment by drets** [comment]
ping @rasendubi

**Comment by rasendubi** [comment]
\o/

File: .travis.yml (hunk 1 above)
Comment (rasendubi) [reply .travis.yml:34]: Would you like to add the comment in the code?

Comment (drets) [reply .travis.yml:34]: Better to add first unit test and enable it.
(I'll try to do it in the next patch)

**Pull request merged**